example/tests/test_helpers_param.py

```diff
@@ -23,4 +26,12 @@ def test_square_parametrized(value, expected):
   ])
 def test_linspace(value, expected):
     # test equality of the two arrays
-    np.array_equal(linspace(*value), np.array(expected))
\ No newline at end of file
+    np.array_equal(linspace(*value), np.array(expected))
+
+
+# Uncomment for "property-based testing" exercises 2-4
+#
+# @given(...)
+# def test_linspace_property(...):
```

Comment:
I was thinking that we should provide maybe a hint about the maximum size of the array (as too large will cause the `hypothesis` to time out or consume too much memory and get killed). But on the other hand perhaps we want the participants to experience that first hand...  could not make up my mind on that point.